doc/gnucobol.texi

```diff
@@ -1749,7 +1754,113 @@ Tracing program execution, either in general or in specific parts can be enabled
 
 @exampleindent 0
 
-@node Extensions, System Routines, Debug, Top
+@node Profiling, Extensions, Debug, Top
+@chapter Profiling COBOL
+@cindex Profiling
+@cindex Profiling your program
+
+@menu
+* Profiling options::           Profiling options
+* Profiling results::           Profiling results
+@end menu
+
+@node Profiling options
+@section Profiling options
+
+Profiling is enabled with the @code{-fprof} flag while compiling
+your COBOL module.
+
+Then executing your program will automatically profile the module(s)
+and generate a CSV result file. By default, this file is called
+@code{<pid>-prof.csv}.
+
+The following environment variables (and the corresponding option in
+the runtime configuration) are available to tune the behavior of
+profiling during execution:
+
+@table @code
+
+@item COB_PROF_FILE (prof_file)
+
+This variable specifies the name of the CSV file generated at the end
+of execution.
+
+@item COB_PROF_DISABLE (prof_disable)
+
+This variable disables profiling for all modules, even the ones
+compiled with @code{-fprof}.
+
+@item COB_PROF_MAX_DEPTH (prof_max_depth)
+
+This variable can be used to increase the maximal number of recursions
```

Comment:
This isn't about recursions, but about "active entries", right?
In any case: please leave the introduction here and for the details refer to the settings annex.

Add those details in config/runtime.cfg to include them in the annex